docs/detections/detection-engine-intro.asciidoc

```diff
@@ -82,7 +82,7 @@ Using cold tier data for unsupported indices may result in detection rule timeou
 [[support-indicator-rules]]
 == Limited support for indicator match rules
 
-Indicator match rules provide a powerful capability to search your security data; however, their queries can consume significant deployment resources. We recommend avoiding indicator index queries that return more indicators than necessary for the desired rule coverage. In addition, the following support restrictions are in place:
+Indicator match rules provide a powerful capability to search your security data; however, their queries can consume significant deployment resources. When creating an <<create-indicator-rule, indicator match rule>>, we recommend limiting the time range of the indicator index query to the minimum period necessary for the desired rule coverage. For example, entering a query such as `@timestamp > "now-30d"` will return indicators ingested during the past 30 days. Without this limitation, the rule will include all of the indicators in your indicator indices, which may cause excessively long queries.
```

Comment:
@nastasha-solomon general question about this: when we say `"may cause excessively long queries"` are we referring to the number of results returned? I want to verify we are using `query` in the right context. Just wanted to double check.

Reply:
@jmikell821 based on @MikePaquette 's response, would the following revision work:
Without this limitation, the rule will include all of the indicators in your indicator indices, which may extend the time it takes for the indicator index query to complete.

Reply:
Inserting the following:
Without this limitation, the rule will include all of the indicators in your indicator indices, which may extend the time it takes for the indicator index query to complete.